docs: polish comms v3 settings (#112)
decentraland/documentation: content/contributor/tutorials/how-to-run-a-catalyst.md

And for file storage + DB storage, an SSD / HDD with 2 Tb capacity should be more than sufficient. At the time of this writing the foundation servers are using a little over 1 Tb for all storage.

## Software pre-requisites

The following are required for running catalyst-owner:

- Linux / MacOS operating system
- Docker / docker-compose
- Git
- LiveKit Cluster 

## Some options to consider before we start

- Do you want your node to be publicly exposed on the Internet? If so, you will need an internet-exposed instance and a public URL to connect to it.
- Is the node going to be used for the development of scenes, wearables, etc? In this case, the hardware specs can be a lot smaller, as it will probably be one or two people accessing it, not hundreds of users.
- Do you want to sync all entity types? If you are using this for developing scenes, probably you want to skip synchronization of profiles, as they take a lot of time, bandwidth and most importantly disk space and won’t benefit you for your goal.

> ## Livekit

Reviewer: ```suggestion
## LiveKit
```

> An external LiveKit cluster is in needed by the Catalyst's Comms service to orchestrate the communications between players. For that, there are two options: use a Livekit Cloud account or run your own Livekit cluster.

Reviewer: ```suggestion
An external LiveKit cluster is in needed by the Catalyst's Comms service to orchestrate the communications between players. For that, there are two options: use a LiveKit Cloud account or run your own LiveKit cluster.
```

> The Decentraland Foundation’s recommendation is to setup [Livekit Cloud account](https://cloud.livekit.io/) which comes with a free tier that can manage 100 users, or paying the service according to the traffic. This approach is easier as it doesn’t require to provision extra infrastructure and the service can manage the scaling.

Reviewer 2: ```suggestion
The recommendation is to setup [Livekit Cloud account](https://cloud.livekit.io/) which comes with a free tier that can manage 100 users, or paying the service according to the traffic. This approach is easier as it doesn’t require to provision extra infrastructure and the service can manage the scaling.
```

> The Decentraland Foundation’s recommendation is to setup [Livekit Cloud account](https://cloud.livekit.io/) which comes with a free tier that can manage 100 users, or paying the service according to the traffic. This approach is easier as it doesn’t require to provision extra infrastructure and the service can manage the scaling.

Reviewer: ```suggestion
The Decentraland Foundation’s recommendation is to setup [LiveKit Cloud account](https://cloud.livekit.io/) which comes with a free tier that can manage 100 users, or paying the service according to the traffic. This approach is the easiest as it doesn’t require provisioning extra infrastructure and the service can manage the scaling.
```

> Otherwise a Livekit cluster will need to be provisioned. LiveKit does a very good job with their documentation:

Reviewer: ```suggestion
Otherwise a LiveKit cluster will need to be provisioned. LiveKit does a very good job with their documentation:
```

- [Deployment](https://docs.livekit.io/oss/deployment/)
- [Distributed Setup](https://docs.livekit.io/oss/deployment/distributed/)

*In case you would like to deep dive on the technical details needed to support LiveKit transport based communications, check out the [ADR-70 New Communications Architecture](https://adr.decentraland.org/adr/ADR-70).*

## Step by step guide

> First thing to do is clone the [catalyst-owner](github.com/decentraland/catalyst-owner) repository from GitHub

Reviewer 2: ```suggestion
First thing to do is clone the [catalyst-owner](https://github.com/decentraland/catalyst-owner) repository from GitHub
```

> cp .env-advanced.example .env-advanced
```

Now using your favorite text editor, make any needed changes in these files. For e.g., setting up the `EMAIL` environment variable with valid email is required so that you can receive updates about Certbot certificate expiration.

Also setting up your particular value for `CATALYST_URL` may be required, especially if your server is going to be exposed publicly on the Internet. For use in your local machine, the default of `http://localhost` will do.

> #### Comms Serivce

Reviewer: ```suggestion
#### Comms Service
```

Once the LiveKit cluster is available you will need to set the specific LiveKit variables for the Comms service to work, e.g.:

```
LIVEKIT_HOST=wss://livekit-1.mydomain.org
LIVEKIT_API_KEY=API-JjHuvM
LIVEKIT_API_SECRET=J7YSHmNzkNCEfT2

> ROOM_PREFIX=node-east-cost1

Reviewer: ```suggestion
ROOM_PREFIX=my-prefix
```

```

> The `ROOM_PREFIX` variable is optional and can be configured to identify livekit rooms created by the catalyst. This is useful if more than one catalyst is using the same Livekit Cluster.

Reviewer: ```suggestion
The `ROOM_PREFIX` variable is optional and can be configured to identify LiveKit rooms created by the Catalyst. This is useful if more than one Catalyst is using the same LiveKit Cluster.
```
